src/routes/cheatsheet/cheat-sheet.js

```diff
@@ -470,5 +470,30 @@ onMount(() => {
   })()
 </script>
 `
-	}
+	},
+  {
+    title: 'Dynamic component',
```

Comment:
all components are themselves dynamic :smile: 
```suggestion
    title: 'Dynamically select a component',
```